rbc/tests/heavydb/test_math.py

```diff
@@ -129,6 +129,9 @@ def test_math_function(heavydb, device, nb_version, fn_name, signature):
     if not heavydb.has_cuda and device == 'gpu':
         pytest.skip('test requires CUDA-enabled heavydb server')
 
+    if not heavydb.has_cuda_libdevice and device == 'gpu':
+        pytest.skip('Test requires CUDA-enabled heavydb server with libdevice')
+
```

Comment:
Update `rbc/tests/test_externals_libdevice.py` as well to skip the tests if libdevice is not available.

Reply:
Fixed.